lib/attachinary/orm/file_mixin.rb

```diff
@@ -10,7 +10,9 @@ def self.included(base)
     end
 
     def as_json(options={})
-      super(only: [:id, :public_id, :format, :version, :resource_type], methods: [:path])
+      super(only: [:id, :public_id, :format, :version, :resource_type], methods: [:path]).merge(
+        preview_url: preview_url(options)
```

Comment:
This looks a little odd, but the `:methods` part of `as_json` won't pass arguments in.